wizeline/wize-q-front
# Feat - WIP-279 - Session token front

#### What does this PR do?
Adds a session token on login that will be send to all api calls in the header, also removes it on logout
#### Where should the reviewer start?
Check src/actions/axiosConfig.js for header change
Check src/auth.js for creation and deletion of the token
Check src/components/EditQuestionModal.js for addition of isAnonymous to update calls
#### How should this be manually tested?
Create a question anonymously, check that you can modify it, modify it; then re-log and you will not be able to modify the previous question
#### Any background context you want to provide?
There is another pull request for back-end that needs to be merged with this one
#### What are the relevant tickets?
[Frontend create the new session hash token](https://wizeline.atlassian.net/browse/WIP-279)
